packages/api-utils/package.json

```diff
@@ -70,8 +71,7 @@
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^3.0.0",
     "graphql": "^15.0.0",
-    "jest": "^26.0.1",
-    "semantic-release": "^17.0.4"
+    "jest": "^26.0.1"
```

Comment:
Are we keeping the version of this consistent?

Reply:
Yeah, but we're touching this one in this PR

Reply:
Sorry, misread the diff. Disregard the comment. But we should do a push to make all common dependencies consistent.